code/game/gamemodes/cult/cult_items.dm

```diff
@@ -41,18 +44,9 @@
 	heat_protection = HEAD
 	max_heat_protection_temperature = HELMET_MAX_TEMP_PROTECT
 
-
-/obj/item/clothing/head/culthood/alt

```

Comment:
Didn't you revert back to using the proper cult sprites? (which confusingly enough, is defined here as the "alt" ones)